backend/src/game/locations.py

```diff
@@ -0,0 +1,6 @@
+from enum import Enum
+
+
+class Locations(Enum):
+    OUTSIDE = 0
```

Comment:
Note that there are 3 locations in the game, but Tokyo and Tokyo Bay are for all intents and purposes the same. I don't think there are any situations where tracking a distinction between Tokyo and Tokyo Bay would be beneficial. While the rules discuss the two locations, it can be simplified to whenever there are more than 4 players, two monsters can be in Tokyo

Reply:
In other words, the absence of Tokyo Bay in the enum makes sense to me, but I don't know if this was the intent of the omission.

Reply:
It actually _does_ matter between Tokyo vs Tokyo Bay, because players move between the two.
I intentionally left that location out because we're not there yet. Let's get a 2 player game to work, first!

Reply:
Ah I re-read some rules and see where it matters now. Oops. Makes sense to focus on 2 player first though.